ras/ras_tools.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2021 IBM
+# Author: Shirisha Ganta <shirisha.ganta1@ibm.com>
+
+from avocado import Test
+from avocado.utils import process
+from avocado.utils.software_manager import SoftwareManager
+
+class Ras_tools(Test):
+    """
+    :avocado: tags=privileged
+    """
+    fail_cmd = list()
+
+    def run_cmd(self, cmd):
+        cmd_result = process.run(cmd, ignore_status=True, sudo=True,
+                                 shell=True)
+        if cmd_result.exit_status != 0:
+            self.fail_cmd.append(cmd)
+        return
+
+    @staticmethod
+    def run_cmd_out(cmd):
+        return process.system_output(cmd, shell=True,
+                                     ignore_status=True,
+                                     sudo=True).decode("utf-8").strip()
+
+    def setUp(self):
+        sm = SoftwareManager()
+        deps = ["ppc64-diag", "powerpc-utils"]
+        for pkg in deps:
+            if not sm.check_installed(pkg) and not sm.install(pkg):
+                self.cancel("Fail to install %s required for this test." %
+                            pkg)
+
+    def test1_nvsetenv(self):
+        self.log.info("===Executing nvsetenv tool====")
+        self.is_fail = 0
+        self.run_cmd("nvsetenv")
+        self.run_cmd("nvsetenv load-base")
+        self.run_cmd("nvsetenv load-base 7000")
```

Comment:
i suggest have these can be part of yml input and like we can have like  
```
value = self.params.get('nvsetenv_list', default=['load-base', 'load-base 7000'])
for list_item in value:
  self.run_cmd('nvsetenv  %s ' % list_item )
  ```

Reply:
okay